provider/block_dirty_bitmap.py

```diff
@@ -25,13 +25,17 @@ def parse_params(vm, params):
         target_image_params = params.object_params(target_image)
         json_backend_list = ["ceph", "iscsi_direct"]
         if target_image_params["image_backend"] in json_backend_list:
+        if "rbd_pool_name" in target_image_params:
```

Comment:
Two if here?

Reply:
target_img_obj = qemu_storage.QemuImg(target_image_params, data_dir.get_data_dir(), target_image)
if target_img_obj.is_remote_image():
get_image_name = qemu_storage.get_image_json

Reply:
Maybe after the conflict was resolved, the code was not pushed. Have updated it to latest. Thanks, zhenchao.

Reply:
still no update here

Reply:
Same comment as bellow.